src/items/class-features/adaptive_metabolism_(ex).json

```diff
@@ -0,0 +1,125 @@
+{
+  "_id": "KSiZ6H88gZuQqzUc",
+  "name": "Adaptive Metabolism (Ex)",
+  "type": "feat",
+  "img": "icons/svg/mystery-man.svg",
+  "system": {
+    "type": "",
+    "ability": "",
+    "abilityMods": {
+      "parts": []
+    },
+    "actionTarget": "",
+    "actionType": "",
+    "activation": {
+      "type": "",
+      "condition": "",
+      "cost": 0
+    },
+    "area": {
+      "effect": "",
+      "shapable": false,
+      "shape": "",
+      "units": "ft",
+      "value": ""
+    },
+    "attackBonus": 0,
+    "chatFlavor": "",
+    "critical": {
+      "effect": "",
+      "parts": []
+    },
+    "damage": {
+      "parts": []
+    },
+    "damageNotes": "",
+    "description": {
+      "chat": "",
+      "gmnotes": "",
+      "short": "",
+      "unidentified": "",
+      "value": "<p>&gt;@UUID[Compendium.sfrpg.class-features.rpaon3jF3nCe5eP9]{Theorem (Ex)}</p>\n<p>As a move action, you can expend one of your biohacks to inject yourself or an ally attuned to your @UUID[Compendium.sfrpg.class-features.lgdDuAM2I3s8PQsD]{custom microlab} with a specially designed compound derived from a native Kehtarian lichen with unsurpassed adaptive properties, momentarily fooling the target’s system into metabolizing chemical compounds differently. Choose any serum with an item level no greater than half your biohacker level. The next poison, drug, or serum that affects the target before the start of your next turn has the effect of the serum you chose instead of its usual one. The substituted effect lasts a maximum of [[/r 1d4]]{1d4 rounds} or the length of the original effect, whichever is shorter. If the original effect is still ongoing when the duration expires, it resumes at the same stage it was at before the change.</p>"
+    },
+    "descriptors": {
+      "acid": false,
+      "air": false,
+      "calling": false,
+      "chaotic": false,
+      "charm": false,
+      "cold": false,
+      "compulsion": false,
+      "creation": false,
+      "curse": false,
+      "darkness": false,
+      "death": false,
+      "disease": false,
+      "earth": false,
+      "electricity": false,
+      "emotion": false,
+      "evil": false,
+      "fear": false,
+      "fire": false,
+      "force": false,
+      "good": false,
+      "healing": false,
+      "language-dependent": false,
+      "lawful": false,
+      "light": false,
+      "mind-affecting": false,
+      "pain": false,
+      "poison": false,
+      "polymorph": false,
+      "radiation": false,
+      "scrying": false,
+      "sense-dependent": false,
+      "shadow": false,
+      "sonic": false,
+      "summoning": false,
+      "teleportation": false,
+      "water": false
+    },
+    "details": {
+      "category": "classFeature",
+      "combat": false,
+      "specialAbilityType": "ex"
+    },
+    "duration": {
+      "units": "text",
+      "value": ""
```

Comment:
You can put "1d4 rounds or the length of the original effect, whichever is shorter" here